### DISCUSSION TITLE
is there a component library on top of unocss?

### DISCUSSION THREAD

**Discussion by duongphuhiep** [unocss/unocss#1179] [Q&A]
I'm looking for a (pre-made) components library which is built on top of unocss, similar to [daisyui](https://daisyui.com/) on top of tailwind.. is it exist?

**Comment by antfu** [comment]
I did made a few experiments here https://github.com/nuxt/ui/tree/main/packages/ui

**Reply by duongphuhiep** [reply]
it is not what I'm looking for:

I'm looking for a bunch of pre-made shortcut: "btn btn-primary", "carousel carousel-item", "drawer drawer-side" etc on top of a the default uno preset. For eg:

![image](https://user-images.githubusercontent.com/1638594/176180553-5d9aa075-d34d-47f1-bfee-4e64674027f5.png)

These "shortcuts" can be use not only for nutx but for any projects (sveltekit, react.. etc)
* so that I don't have to wrote them again for each project, and
* beginners (like me) can have something nice for quick prototyping.. and still get the power to tweak whatever we needed to later on

**Comment by ManUtopiK** [comment]
[Anu-vue](https://anu-vue.netlify.app) is an UI library for Vue that use unocss and vueUse.

**Comment by husayt** [comment]
Here are few options:

* [Anu-vue](https://anu-vue.netlify.app/)
* [Onu UI](https://onu.zyob.top/)

**Comment by polyzen** [comment]
> similar to [daisyui](https://daisyui.com/) on top of tailwind..

https://unocss-preset-daisy.vercel.app/
~~Though there's this issue: https://github.com/kidonng/daisyui/issues/1~~

Also https://github.com/johannschopplich/duecss

**Comment by StarLederer** [comment]
Since you said you are not looking for multi-element components built for UI frameworks like Vue or React in reply to other suggestions it sounds like you are looking for an UnoCSS port of [picocss](https://picocss.com/). I don't think something like that exists but perhaps you can be the one who makes the port?

**Comment by s3xysteak** [comment]
[daisyui](https://daisyui.com/) looks so good, however it has [no plans to support unocss](https://github.com/saadeghi/daisyui/issues/650) 😢 .

 There is a [preset](https://github.com/kidonng/unocss-preset-daisy) to make `daisyui` and `unocss` work together. But it seems the repo is inactive because the maintainer has no response for months. It not work with `daisyui - v4`

**Reply by Johnkat-Mj** [reply]
Hello, we're currently working on something like DaisyUI for UnoCss, it will be published on npm today. 
Check it [here](https://github.com/unify-ui-dev/unify-preset)

**Comment by ManUtopiK** [comment]
For one side project, I'm using [devtools-ui-kit](https://github.com/nuxt/devtools/tree/main/packages/devtools-ui-kit) from the nuxt devtools. It was built by Antfu him-self ! Some components are specific for the devtools, but you can use the power of unocss with the `n` props. Ex: `n="solid borderless red"`. It's quite useful. I think this library is very specific and can be used for micro-applications. I not recommend for a large app, but why not !
